.build-tools/pkg/metadataschema/builtin-authentication-profiles.go

```diff
@@ -20,6 +20,68 @@ import (
 // ParseBuiltinAuthenticationProfile returns an AuthenticationProfile(s) from a given BuiltinAuthenticationProfile.
 func ParseBuiltinAuthenticationProfile(bi BuiltinAuthenticationProfile) ([]AuthenticationProfile, error) {
 	switch bi.Name {
+	case "aws":
+		return []AuthenticationProfile{
+			{
+				Title:       "AWS: Access Key ID and Secret Access Key Dapr Metadata",
+				Description: "Authenticate using an Access Key ID and Secret Access Key included in the metadata",
+				Metadata: []Metadata{
+					{
+						Name:        "AWS_ACCESS_KEY_ID",
```

Comment:
According to the docs, shouldn't these be `accessKey` and `secretKey`? https://docs.dapr.io/reference/components-reference/supported-bindings/s3/